AEPMessaging/Tests/UnitTests/MessagingPropertiesTests.swift

```diff
@@ -0,0 +1,94 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import XCTest
+@testable import AEPMessaging
+import AEPServices
+
+class MessagingPropertiesTests: XCTestCase {
```

Comment:
Add a test which sets a pushIdentifier value, then sets a different value verifying the MessagingProperties local and persisted values are the new value.